docs/modules/ic-identity-guide/pages/auth-how-to.adoc

```diff
@@ -67,11 +69,13 @@ On the next screen, you can select one of the following options:
 
 * *Seed Phrase*
 + 
-Select this option to generate a cryptographically-secure seed-phrase that you can use to recover your account. Make sure you save this phrase somewhere safe and known only to you. *Note that the first string in your seed phrase is your ID number*. You will need this number to begin the the recovery process.
+Select this option to generate a cryptographically-secure seed-phrase that you can use to recover your account. Make sure you save this phrase somewhere safe and known only to you. *Note that the first string in your seed phrase is your user number*. You will need this number to begin the the recovery process.
```

Comment:
See above.

Reply:
I'm not sure if this references test in the UI or not.